repos/spack_repo/builtin/packages/latte/package.py

```diff
@@ -46,13 +52,21 @@ def cmake_args(self):
         else:
             options.append("-DBUILD_SHARED_LIBS=OFF")
         if self.spec.satisfies("+mpi"):
-            options.append("-DO_MPI=yes")
+            options.append("-DDO_MPI=ON")
+            options.append("-DCMAKE_C_COMPILER=%s" % self.spec["mpi"].mpicc)
+            options.append("-DCMAKE_CXX_COMPILER=%s" % self.spec["mpi"].mpicxx)
+            options.append("-DCMAKE_Fortran_COMPILER=%s" % self.spec["mpi"].mpifc)
```

Comment:
Nit. You could be using python f-strings here and elsewhere (see https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals).

Reply:
Not a problem. We've been switching back core to use F strings and are encouraging packaged developers to do the same.